lib/config/presets/internal/group.ts

```diff
@@ -200,6 +200,19 @@ const staticGroups = {
       },
     ],
   },
+  gradle: {
+    description:
+      "Group anything that looks like Gradle together so that it's updated together.",
+    packageRules: [
+      {
+        commitMessageTopic: 'Gradle',
+        matchDatasources: ['docker', 'gradle-version'],
+        matchPackageNames: [
+          '/(?:^|/)gradle/', // gradle or ends with "/gradle"
```

Comment:
```suggestion
          '/(?:^|/)gradle$/', // gradle or ends with "/gradle"
```
otherwise it will allow `something/gradle124node`